forward flags support for filter_var in fetchGlobal ( and all helper func )

puschie286 opened the pull request:
getVar, getGet, getPost, getPostGet, getGetPost, getCookie, getEnv, getServer

lonnieezell commented:
These changes look good, but a few things are needed: 

1. We ask that PR's are [GPG signed](https://bcit-ci.github.io/CodeIgniter4/contributing/signing.html). 
2. Needs updated docs
3. Needs tests in place for all of this...

puschie286 commented:
sry but i tired several times to push the commit and end up with losing all of my changes ( even pushed commits are lost .... ) dont know how this happend. so i will close it, maybe someone else will do this

puschie286 commented:
so i got fixed it - sry for all that confusion

i just updated the functions in the guide ( and added a link to the filter flag php site ) - i hope someone else ( with better english knowledge ) could write a example usage
